Content.Shared/_Moffstation/Weapons/Ranged/Components/RecoilKickSusceptibleComponent.cs

```diff
@@ -0,0 +1,32 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._Moffstation.Weapons.Ranged.Components;
+
+[DataDefinition]
+public sealed partial class RecoilKick
+{
+    /// <summary>
+    /// The time (in seconds) a wielder will be airborne when affected by recoil kick.
+    /// </summary>
+    public const float FlyTime = 0.2f;
+
+    /// <summary>
+    /// The impulse (in Newton-seconds) this gun applies to wielders who are
+    /// <see cref="RecoilKickSusceptibleComponent">susceptible to recoil kick</see> when it is fired.
+    /// </summary>
+    [DataField]
+    public float Impulse;
```

Comment:
The physical action applied to the wielder, this should be about 25 for a Mosin. Other guns can figure it out from there.

I think it'd be neat if different ammunition could modify this (eg. practice shoots less mass, so less kick; uranium shoots more mass so more kick. Nox can tell me if this is realistic or not)